app/components/nav-header.tsx

```diff
@@ -24,27 +29,37 @@ export function NavHeader() {
 					className={clsx(
 						"flex text-sm font-mono border rounded-lg divide-x dark:(border-neutral-700 divide-neutral-700) overflow-hidden",
 						{
-							"!border-blue-500/70": isLogs,
-							"!border-rose-500/70": isRequests,
+							"!border-blue-500/70": isLogs && appCount > 0 && appId,
```

Comment:
```suggestion
							"!border-blue-500/70": isLogs && appId,
```
Should be enough or?

Reply:
Correct